recipes/mmlong2-lite/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+mkdir -p $PREFIX/bin/ 
+cp -r src/* $PREFIX/bin/.
```

Comment:
what's the purpose of the trailing dot here ?

Reply:
The trailing dot is unnecessary